bin/start-uc-cli-in-docker

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
```

Comment:
why would we want to start the uc cli in docker?

Reply:
We don't need to, but if you want to have the client connect to a remote UC server, but you don't want to build the client on your machine, then docker is a way to run the client inside a container.

Reply:
Actually quite common approach jean is taking there. I support the idea.

Reply:
@vikrantpuppala Can I mark this as resolved or would you like me to remove the docker CLI piece ?

I think it's a nice to have especially for experimentation, or for someone who is interested in building only the CLI in docker to connect to a remote server.